### DISCUSSION TITLE
Enhancement: Add Winning Icons for when the opponent or player is winning

### DISCUSSION THREAD

**Discussion by Jaduplex** [FunkinCrew/Funkin#1582] [Ideas]
I WAS going to do a pull request for this, but I really don't know anything about HaxeFlixel so I guess a suggestion will have to do. Essentially, I'm suggesting that, when the opponent or player is losing, the player or opponent that is winning against them has an excited or happy icon. Also, maybe add a losing icon for Girlfriend in the tutorial?

**Comment by isakube** [comment]
and the gf losing would be a ^^
